telegram/ext/filters.py

```diff
@@ -603,6 +603,45 @@ def filter(self, message: Message) -> bool:
         xml = mime_type('application/xml')
         zip = mime_type('application/zip')
 
+        class file_extension(MessageFilter):
+            """This filter filters documents by their file ending/extension.
+
+            Note:
+                This Filter only filters by the file ending/extension of the document,
+                    it doesn't check the validity of document.
+                The user can manipulate the file extension of a document and
+                    send media with wrong types that don't fit to this handler.
+
+            Example:
+                ``Filters.document.file_extension('jpg')`` filters files with extension ``.jpg``.
+            """
+
+            file_extension: Optional[List[str]] = None
```

Comment:
What is this good for?

Reply:
To not assing the attr in `__init__()` if it's None. Not much good, i'll might edit this.

Reply:
please do ;)